File: App.Admin/src/api/services/table/models/EditTableColumnConfigInput.ts
import { FaTableColumnCtx } from "./FaTableColumnCtx";

/**
 * Fast.Center.Service.Table.Dto.EditTableColumnConfigInput 编辑表格列配置输入
 */
export interface EditTableColumnConfigInput {
  /**
   * 表格Id
   */
  tableId?: number;
  /**
   * 更新版本控制字段
   */
  rowVersion?: number;
  /**
   * 表格列
   */
  columns?: Array<FaTableColumnCtx>;
}

File: Api.Server/src/Services/Center/Table/Dto/EditTableColumnConfigInput.cs
namespace Fast.Center.Service.Table.Dto;

/// <summary>
/// <see cref="EditTableColumnConfigInput"/> 编辑表格列配置输入
/// </summary>
public class EditTableColumnConfigInput : UpdateVersionInput
{
    /// <summary>
    /// 表格Id
    /// </summary>
    [LongRequired(ErrorMessage = "表格Id不能为空")]
    public long TableId { get; set; }

    /// <summary>
    /// 表格列
    /// </summary>
    public List<FaTableColumnCtx> Columns { get; set; }
}